server/utils/applications/applicationIdentityBar.test.ts

```diff
@@ -34,12 +50,18 @@ describe('applicationIdentityBar', () => {
       const person = personFactory.build()
       const application = applicationFactory.build({ person, status: 'withdrawn' })
 
+      config.flags.oneApplication = true
       expect(applicationTitle(application, 'heading')).toMatchStringIgnoringWhitespace(`
-        <h1 class="govuk-caption-l">heading</h1>
-        <h2 class="govuk-heading-l">
-          ${displayName(person)}
+        <h1 class="govuk-heading-l">
+          heading
           <strong class="govuk-tag govuk-tag--red govuk-!-margin-5" data-cy-status="withdrawn">Application withdrawn</strong>
```

Comment:
The status tags are very wordy: given the context of an application is always rather clear, `Withdrawn` should be enough, instead of `Application withdrawn`. Not one for this PR, though!